packages/process/_test.pony

```diff
@@ -95,6 +120,39 @@ class iso _TestFileExec is UnitTest
   fun timed_out(h: TestHelper) =>
     h.complete(false)
 
+  fun _setup_notifier(h: TestHelper, path: FilePath): ProcessNotify iso^ =>
+    recover object is ProcessNotify
```

Comment:
I think you can spare yourself the extra `end` for the `recover` if you use `object iso` instead of `recover object` here.  It's just a minor style point, but I thought you might want to know.